src/Pagination/index.jsx

```diff
@@ -343,6 +345,9 @@ class Pagination extends React.Component {
     });
 
     return pages.map((pageIndex) => {
+      if (pageCount <= 1) {
+        return null;
+      }
```

Comment:
[curious] should we even need to iterate through `pages` array if `pageCount <= 1`?